Utils/github_workflow_scripts/close_stale_contrib_branches.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python3
+
+import os
+import sys
+import urllib3
+import inspect
+from blessings import Terminal
+from github import Github, enable_console_debug_logging
+from github.Repository import Repository
+from typing import List
+from dateparser import parse
+from datetime import datetime, timezone
+from utils import timestamped_print
+
+urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
+print = timestamped_print
+
+
+def get_stale_branch_names_with_contrib(repo: Repository) -> List[str]:  # noqa: E999
+    """Return the list of branches that have the prefix of "contrib/" without open pull requests
+    and that have not been updated for 2 months (stale)
+
+    Args:
+        repo (Repository): The repository whose branches will be searched and listed
+
+    Returns:
+        (List[str]): List of branch names that are stale and have the "contrib/" prefix
+    """
+    now = datetime.now(timezone.min)
+    organization = 'demisto'
+    branch_names = []
+    for branch in repo.get_branches():
+        # Make sure the branch is contrib
+        if branch.name.startswith('contrib/'):
+            prs_with_branch_as_base = repo.get_pulls(state='OPEN', base=branch.name)
+            prs_with_branch_as_head = repo.get_pulls(state='OPEN', head=f'{organization}:{branch.name})')
+
+            # Make sure there are no open prs pointing to/from the branch
+            if prs_with_branch_as_base.totalCount < 1 and prs_with_branch_as_head.totalCount < 1:
+                # load all members
+                inspect.getmembers(branch.commit)
```

Comment:
This seems to me like something that you used during development and should be removed for production (also the import).

Reply:
This is actually intentional - the members of `branch.commit` are supposed to be lazily loaded, but they are failing to load when calling the `branch.commit` directly - so this is a workaround to make it load its members.

Reply:
I don't think there is a need for this. Instead of using `branch.commit.last_modified`, try using `branch.commit.commit.last_modified`. I think that should resolve that issue.